test/Plutip.purs

```diff
@@ -177,6 +183,55 @@ suite = do
             liftedE $ balanceAndSignTxE ubTx
           submitAndLog bsTx
 
+    test "runPlutipContract: parallel Pkh2Pkh" do
+      let
+        distribution :: InitialUTxO /\ InitialUTxO
+        distribution =
+          [ BigInt.fromInt 1_000_000_000
+          , BigInt.fromInt 2_000_000_000
+          ] /\
+            [ BigInt.fromInt 1_000_000_000
+            , BigInt.fromInt 2_000_000_000
+            ]
+      withPlutipContractEnv config distribution \env (alice /\ bob) -> do
```

Comment:
Instead of manually forking `Aff`, we could use the `Parallel` class:

```purescript
  do
    ...
    withPlutipContractEnv config distribution \env (alice /\ bob) -> sequential ado
      parallel $ runContractInEnv env $ withKeyWallet alice do
        ...
      parallel $ runContractInEnv env $ withKeyWallet bob do
        ...
      in unit
```

For tests it doesn't matter as much, but I suppose this acts as an example, and users are probably better off using the parallel interface?

Reply:
Agreed